hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/googlepubsub/GooglePubSubSenderTarget.java

```diff
@@ -1,27 +1,38 @@
 package pl.allegro.tech.hermes.consumers.consumer.sender.googlepubsub;
 
 import com.google.pubsub.v1.TopicName;
+import pl.allegro.tech.hermes.consumers.consumer.sender.googlepubsub.transformer.compression.CompressionCodec;
 
 import java.util.Objects;
 
-class GooglePubSubSenderTarget {
+public class GooglePubSubSenderTarget {
```

Comment:
So far, the `googlepubsub` package is relatively small in terms of the number of classes it has. Therefore, to avoid making this class public, I would suggest not to create additional subpackages (`transformer` and `compression`).

Reply:
OK, moving classes to the `googlepubsub` and restricting visibility if possible.